### PR TITLE
Resprites chemical bottles

### PR DESCRIPTION
**Pull request by DisturbHerb**
## About the PR 
Resprited the chemical bottles that come in Syndicate poison bottle kits as well as medicines from the Nanomed. They're funky now, and have coloured caps depending on the reagent they start with. They also have differing underlays based on fluid levels, like beakers. Poggers!
image

Sprite comparisons woo:
chem_bottles

## Why's this needed? 
Old sprite flat. New sprite _round._ Ooga booga.

### CODE REVIEW

**Comment by Alexalmighty502** [comment]
I'm worried that this will make it difficult to quickly judge how many units you have left just from sight alone.

**Comment by DisturbHerb** [comment]
> I'm worried that this will make it difficult to quickly judge how many units you have left just from sight alone.

Personally, I felt that it was difficult enough with the old sprites, but I think that could just be me.

**Comment by MatheusIsNotAvailable** [comment]
They look like they have almost nothing in them, specially the first one, even though they are full.

**Comment by DisturbHerb** [comment]
Made some changes per Matheus' suggestions.

image
image

**Comment by DisturbHerb** [comment]
even newer changes. they now gots fluid levels. original post edited

File: code/modules/chemistry/tools/bottles.dm [hunk 1]
```diff
@@ -19,17 +20,28 @@
 	New()
 		if (!src.bottle_style)
 			src.bottle_style = "[rand(1,4)]"
-		..()
+		..() // i have no damn clue why, but ..() *has* to be here otherwise it breaks everything about the damn cap colours wHY DID I DO THIS - DisturbHerb
+		src.cap_image = image('icons/obj/chemical.dmi', "bottle[bottle_style]-cap", -1)
+		if (src.reagents)
+			var/list/color_HSL = rgb2num((reagents.get_average_color()).to_rgb(), COLORSPACE_HSL)
+			if (color_HSL[3] < 80)
+				color_HSL[3] = 80
```
Comment (ZeWaka) [inline code/modules/chemistry/tools/bottles.dm:28]:
```suggestion
			color_HSL[3] = max(80, color_HSL[3]) // minimum lightness
```

**Review by ZeWaka — CHANGES_REQUESTED** [review]

**Comment by DisturbHerb** [comment]
Closing so I can figure out what to do with this later.

**Pull request closed without merging**